appeals/web/src/server/appeals/appeal-details/neighbouring-sites/neighbouring-sites.service.js

```diff
@@ -20,3 +19,49 @@ export function addNeighbouringSite(apiClient, appealId, neighbouringSite) {
 		}
 	});
 }
+
+/**
+ *
+ * @param {import('got').Got} apiClient
+ * @param {string} appealId
+ * @param {import('@pins/appeals.api').Appeals.AppealSite} neighbouringSite
+ * @param {string} siteId
+ * @returns {Promise<{}>}
+ */
+export function changeNeighbouringSite(apiClient, appealId, neighbouringSite, siteId) {
+	const formattedNeighbouringSite = {
+		siteId: parseInt(siteId),
```

Comment:
Very minor, just wondering if there's any chance this parseInt could fail?

Reply:
The data as it is on the server  says siteId is a number it's just converted from int to string when creating the url so this reverts it back